releasewarrior/templates/firefox/release-rc.json.tmpl

```diff
@@ -28,30 +34,30 @@
                 "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#obtain-sign-offs-for-changes",
                 "resolved": false
             },
-            {
-                "alias": "wnp",
-                "description": "setup whatsnew page",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md",
-                "resolved": false
-            },
             {
                 "alias": "mirrors",
                 "description": "pushed to mirrors/releases",
                 "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md#push",
                 "resolved": false
             },
+            {
+                "alias": "remove_old_wnp",
+                "description": "remove previous version's What's New Page",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md#remove-wnp",
+                "resolved": false
+            },
             {
                 "alias": "ship",
-                "description": "publish release tasks",
+                "description": "publish release",
                 "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md#ship",
                 "resolved": false
-             },
-             {
+            },
+            {
                 "alias": "signoff",
                 "description": "signoff in Balrog",
                 "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#obtain-sign-offs-for-changes",
```

Comment:
This needs to point to https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md#obtain-sign-offs-for-changes instead